tests/SymfonyBundle/DependencyInjection/Compiler/ConfigureMiddlewaresTest.php

```diff
@@ -35,41 +36,57 @@ protected function setUp()
      */
     public function it_configures_a_chain_of_buses_according_to_the_given_priorities()
     {
-        $this->createBusDefinition('middleware100', 100);
-        $this->createBusDefinition('middleware-100', -100);
-        $this->createBusDefinition('middleware200', 200);
+        $classes = [
+            'SimpleBus\SymfonyBridge\Tests\Functional\SmokeTest\Auto\AuteEvent1' => 100,
```

Comment:
Typo `AuteEvent1` > `AutoEvent1 `

It's better to use `AutoEvent1::class`. This project is PHP 5.6+ so it's supported.

Reply:
Sure. I'll make the changes.